Dockerfile

```diff
@@ -0,0 +1,35 @@
+# use ubuntu as the base image; install R and Python on top
+FROM ubuntu:latest
+
+# avoid humna input for geography and stuff
+ENV DEBIAN_FRONTEND=noninteractive
+
+# install R and python
+RUN apt-get update && apt-get install -y --no-install-recommends build-essential r-base python3.7 python3-pip python3-setuptools python3-dev git
+
+# copy requirements over to application
+COPY requirements.txt /multivac/requirements.txt
+
+WORKDIR /multivac
+
+# set up bdist_wheel
+RUN pip3 install wheel
+
+RUN pip3 install setuptools
+
+# env setup for tensorflow
+RUN pip3 install -r requirements.txt
+
+RUN git clone https://github.com/thunlp/OpenKE && cd OpenKE && sh make.sh
+
+COPY . /multivac
+
```

Comment:
Question on this - what does this directory structure look like after this? I believe all of our import references to OpenKE use stuff like `from OpenKE import ...` and not `from multivac.OpenKE import ...`

Reply:
This will be fine if your imports are like this, OpenKE will be at the root level of `multivac`.